installation/from_source_repository.md

```diff
@@ -1,23 +1,23 @@
 # From sources
 
-If you want to contribute then you might want to install Crystal from sources. But Crystal is written in Crystal itself! So you first need to use one of the previous described methods to have a running compiler.
+If you want to contribute then you might want to install Crystal from sources.
 
-You will also need LLVM 3.5 or 3.6 present in the path. If you are using Mac and the Homebrew formula, this will be automatically configured for you if you install Crystal adding `--with-llvm` flag.
+1. [Install the latest Crystal release](https://crystal-lang.org/docs/installation). To compile Crystal, you need Crystal :).
 
+2. Make sure LLVM 3.5 or 3.6 is present in the path. If you are using Mac and the Homebrew formula, this will be automatically configured for you if you install Crystal adding `--with-llvm` flag.
```

Comment:
This is a bit out of date. We already support LLVM 3.9 and we should encourage new users to use it if possible. So I'd suggest:

"Make sure a supported LLVM version is present in the path. Currently, Crystal supports LLVM 3.5, 3.6, 3.8 and 3.9. When possible, use the latest one. If you are using Mac [...]"